src/App.js

```diff
@@ -1,24 +1,122 @@
-import logo from './logo.svg';
+/* import axios from 'axios';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 import './App.css';
+import './styles/Navbar.scss';
+import Reservations from './components/Reservations';
+import Navbar from './components/navbar';
+import Home from './components/home';
+import DeleteRooms from './components/DeleteRooms';
+import AddReservation from './components/AddReservation';
+import Room from './components/Room';
+import LoginForm from './components/LoginForm';
+
+const token = localStorage.getItem('yourTokenKey'); // Retrieve the token from where you stored it
+
+if (token) {
+  axios.defaults.headers.common.authorization = `Bearer ${token}`;
+  console.log(axios.defaults.headers.common.authorization);
+}
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <Provider store={store}>
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="rooms/:id" element={<Room />} />
+          <Route path="/reservations" element={<Reservations />} />
+          <Route path="/addReservation" element={<AddReservation />} />
+          <Route path="/deleteRooms" element={<DeleteRooms />} />
+          <Route path="/loginForm" element={<LoginForm />} />
+        </Routes>
+      </Provider>
+    </BrowserRouter>
+  );
+}
```

Comment:
- [ ] Kindly, remove this old commented code, in order to improve the repo's cleanliness. 👍